lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -4,8 +4,8 @@ class GlobalCollectGateway < Gateway
       self.display_name = 'GlobalCollect'
       self.homepage_url = 'http://www.globalcollect.com/'
 
-      self.test_url = 'https://eu.sandbox.api-ingenico.com/'
-      self.live_url = 'https://api.globalcollect.com/'
+      self.test_url = 'https://eu.sandbox.api-ingenico.com'
+      self.live_url = 'https://api.globalcollect.com'
```

Comment:
Is the removal of the `/` intentional?

Reply:
Yeah; it _worked_, but was doing double `/`s in the URL. Check out [how we build the URL](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/global_collect.rb#L217) for why.

Reply:
Alrighty.